cdc/model/sink.go

```diff
@@ -690,6 +690,9 @@ type DDLEvent struct {
 	Charset      string           `msg:"-"`
 	Collate      string           `msg:"-"`
 	IsBootstrap  bool             `msg:"-"`
+	// BDRRole is the role of the TiDB cluster, it is used to determine whether
+	// the DDL is executed by the primary cluster.
+	BDRRole string `msg:"-"`
```

Comment:
How the `BDRRole` is set ?

Reply:
It is set by the upstream TiDB and will store in the `DDLJob` struct.